src/overlays/misc/ovl_kaleido_scope/z_lmap_mark.c

```diff
@@ -127,7 +127,13 @@ void PauseMapMark_DrawForDungeon(PlayState* play) {
                                     G_TX_NOMIRROR | G_TX_WRAP, G_TX_NOMASK, G_TX_NOMASK, G_TX_NOLOD, G_TX_NOLOD);
 
                 Matrix_Push();
+
+#if OOT_DEBUG
                 Matrix_Translate(GREG(92) + markPoint->x, GREG(93) + markPoint->y, 0.0f, MTXMODE_APPLY);
+#else
```

Comment:
small thing, but if it matches, i think it makes more sense written as markpoint + reg, so that the reg is an offset of the normal value